packages/astro-embed-youtube/YouTube.astro

```diff
@@ -45,7 +45,7 @@ const href = `https://youtube.com/watch?v=${videoid}`;
 	{...attrs}
 	style={`background-image: url('${posterURL}');`}
 >
-	<a {href} class="lty-playbtn">
+	<a {href} class="lty-playbtn" role="link" aria-label={title}>
 		<span class="lyt-visually-hidden">{attrs.playlabel}</span>
```

Comment:
Thanks for the PR!

This looks like it’s actually a bug in the use of `attrs.playLabel` as a child of the link. As documented in the [`<YouTube>` component docs](https://astro-embed.netlify.app/components/youtube/#playlabel), this should have a default value of `"Play"`, but it looks like we’re not providing that default, which we should do here:
```suggestion
	<a {href} class="lty-playbtn">
		<span class="lyt-visually-hidden">{attrs.playlabel || 'Play'}</span>
```

In general, actually text content is preferable to `aria-label` as it is more widely accessible. There is also no need to apply a `role="link"` to an `<a>` element as this is already the default role.

Reply:
yeah, you're right if i provide `playlabel` that accessiblity issue gone, so i'll add default value now as you suggested.